server/src/test/java/org/elasticsearch/index/shard/IndexShardTests.java

```diff
@@ -2962,13 +2979,13 @@ public void testSegmentMemoryTrackedWithRandomSearchers() throws Exception {
         for (Thread t : threads) {
             t.join();
         }
+        // Close remaining searchers
+        IOUtils.close(searchers);
```

Comment:
Why was this move earlier? I think I miss something I don't know - can you explain?